### PR TITLE
[Observer] Add script and docs for speaker voice management

### PR DESCRIPTION
**Pull request by caisq**
with Azure Cognitive Speech Speaker ID service (text-independent).

### CODE REVIEW

**Comment by caisq** [comment]
cc @mpbrenner Feel free to review or comment on this PR.

**Review by vsubhashini — APPROVED** [review]
Cool!

File: Observer/SpeakFasterObserver Decoder/speaker_id_profiles.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      _get_azure_endpoint(region))
+  session = requests.Session()
+  resp = session.post(url, headers=headers, data=data)
+  resp_json = json.loads(resp.content)
+  profile_id = resp_json["profileId"]
+  print("New speaker profile ID: %s" % profile_id)
+  return profile_id
+
+
+MIN_WAV_LENGTH_SECONDS = 20
```
Comment (TeamGleasonDev) [inline Observer/SpeakFasterObserver Decoder/speaker_id_profiles.py:62]: Consts to the top of the file?

File: Observer/SpeakFasterObserver Decoder/README.md [hunk 2]
```diff
@@ -197,3 +197,40 @@ When you run into these errors, go back to ELAN, fix the problem and re-export
 the `curated.tsv` file and re-run the `elan_process_curated.tsv`. Fix all problem
 until the script says "Success..." and exports a file in the same directory named
 `curated_processed.tsv`. This new TSV file is ready for data ingestion.
+
```
Comment (TeamGleasonDev) [inline Observer/SpeakFasterObserver Decoder/README.md:200]: I'm not familar with using Azure cloud speech, but is there an existing equivalent to this functionality in the az-cli tool?

https://docs.microsoft.com/en-us/cli/azure/

Comment (TeamGleasonDev) [reply Observer/SpeakFasterObserver Decoder/README.md:200]: The cognitive services functionality in az-cli seems to be pretty thin...

https://docs.microsoft.com/en-us/cli/azure/cognitiveservices?view=azure-cli-latest

**Review by TeamGleasonDev — APPROVED** [review]

File: Observer/SpeakFasterObserver Decoder/speaker_id_profiles.py (hunk 1 above)
Comment (caisq) [reply Observer/SpeakFasterObserver Decoder/speaker_id_profiles.py:62]: Done.

File: Observer/SpeakFasterObserver Decoder/README.md (hunk 2 above)
Comment (caisq) [reply Observer/SpeakFasterObserver Decoder/README.md:200]: Right, az-cli doesn't have speech-related features. There is a separate Speech CLI, but it only has speech recognition and TTS, but nothing related to speaker diarization or speaker recognition.

https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/spx-basics?tabs=windowsinstall

**Review by caisq — COMMENTED** [review]
Thanks for the review.

**Pull request merged**